scripts/build_review_app.sh

```diff
@@ -40,29 +40,15 @@ hokusai registry push --force --skip-latest --overwrite --verbose --tag "$NAME"
 # Edit the K8S YAML to reference the proper Docker image
 sed -i.bak "s/:staging/:$NAME/g" "$review_app_file_path" && rm "$review_app_file_path.bak"
 
-# Edit the K8S YAML to remove the instructions that enforce that the service
-# can only be accessible via Cloudflare
-#
-# First, remove the `loadBalancerSourceRanges:` line
-sed -i.bak '/loadBalancer/d' "$review_app_file_path" && rm "$review_app_file_path.bak"
-
-# Then, delete all the IP address lines.
-#
-# WARNING: This is a bit sketchy as this will
-# delete any line of the form "- [number][number][number].". This is my best
-# approx for an regex for an IP address, and I'm sure that there are better
-# ones.
-sed -i.bak "/- [[:digit:]][[:digit:]][[:digit:]]./d" "$review_app_file_path" && rm "$review_app_file_path.bak"
+# Edit the K8S YAML Ingress resource to use the Review App's name as the host.
+sed -i.bak "s/host: staging.artsy.net/host: $NAME.artsy.net/g" "$review_app_file_path" && rm "$review_app_file_path.bak"
```

Comment:
This is unrelated to ingress, but it would be nice if all review apps were identifiable as such, with a prefix like `review-*`. This is an easy place to accomplish that (in coordination with a slight doc update above).

But that's just an idea. It's fine to leave this as-is if we're hoping to switch to sub-subdomains in the near term anyway.

Reply:
I like that idea. We can re-visit this when we are decided on review.artsy.net or not.